Gruntfile.js

```diff
@@ -48,25 +52,40 @@ module.exports = function(grunt) {
           open: true,
         }
       }
+    },
+    concat: {
+        basic: {
+          src: ['src/js/DncApp.js'],
+          dest: 'dist/js/dnc.js'
+        },
+        extras: {
+          src: ['src/js/L.DNC.js', 'src/js/**/*.js'],
+          dest: 'dist/js/L.DNC.js'
```

Comment:
I'm a bit confused by this.  What's the difference between `dnc.js` and `L.DNC.js`?